tests/functional/docs/test_help_output.py

```diff
@@ -432,3 +433,54 @@ def test_operation_help_command_has_note(self):
         self.assert_contains(
             'The default endpoint data.iot.[region].amazonaws.com is '
             'intended for testing purposes only.')
+
+
+class TestAliases(BaseAWSHelpOutputTest):
+    def setUp(self):
+        super(TestAliases, self).setUp()
+        self.files = FileCreator()
+        self.alias_file = self.files.create_file('alias', '[toplevel]\n')
+
+        # TODO: If an environment variable is ever exposed to
```

Comment:
Really did not like that I had to do this. My only other option would have been to expose an environment variable or additional configuration. I am up for ideas on this.